extensions/wikia/UserLogin/js/UserLoginAjaxForm.js

```diff
@@ -87,8 +96,15 @@
 	 * @param {string} html
 	 */
 	UserLoginAjaxForm.prototype.retrieveTemplateCallback = function (html) {
-		var content = $('<div>').hide().append(html),
-			form = this.form;
+		var content, form;
+
+		if (typeof this.options.retrieveTemplateCallback === 'function') {
```

Comment:
My suggestion:
I wouldn't do this check. It may be not politically correct (or it might be that I don't have enough context :) ), but if someone are using this extension, and trying pass to _options.retrieveTemplateCallback_ something that is not a function, then he is asking for troubles, or more probably he is doing some mistake. It would be helpful for that person if this code will blow up - impossible to miss, and easy to understand why.